### PR TITLE
New full sized nav bar with feedback

### LINKED ISSUE #114: Need a feedback link in all the pages
We need to add a feedback link on all the pages so that user don't forget to add a feedback

### PR DESCRIPTION
**Pull request by ssani7**
fix #114

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,43 +1,158 @@
...
+      else setSolidNav(false)
+    }
+
+    document.addEventListener('scroll', handleScroll)
+    return () => {
+      document.removeEventListener('scroll', handleScroll)
+    }
+  }, [])
+
+  const navigaeToPage = () =>{
```
Comment (cuteolaf) [inline src/components/Navbar.tsx:68]: a typo here

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,43 +1,158 @@
...
+      }
+    };
+    setup();
+  }, []);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const isScrolled = window.scrollY > 50
+
+      if (isScrolled) setSolidNav(true)
```
Comment (cuteolaf) [inline src/components/Navbar.tsx:58]: setSolidNav(isScrolled);

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -1,43 +1,158 @@
...
+
+
   return (
     <>
-      <header className="py-[15px] lg:py-[30px]" id="header-wrapper">
-        <div id="main-header">
+      <header className={`navBar ${solidNav ? "bg-theme-black-text" : "bg-transparent"}`} id="header-wrapper">
+        <div className="text-center w-full bg-primary text-black py-1 text-xs lg:text-sm">
+          Thanks for trying the beta version of Imbue. Please let us know what we should work on to make it better! Submit your feedback
+          <a href="https://pfljr3ser45.typeform.com/to/bv00pviH" target="_blank" className="underline cursor-pointer ml-1">here</a>
```
Comment (cuteolaf) [inline src/components/Navbar.tsx:83]: `https://pfljr3ser45.typeform.com/to/bv00pviH`

this link should be configurable.

File: src/components/Navbar.tsx [hunk 4]
```diff
@@ -1,43 +1,158 @@
...
+        </Menu>
       </header>
-      <div
-        onClick={toggleSideBar}
-        className={`modal ${sideBarIsVisivle ? "show" : ""}`}
-        id="modal"
+      <Login
+        visible={loginModal}
+        setVisible={(val) => {
+          console.log("object");
```
Comment (cuteolaf) [inline src/components/Navbar.tsx:151]: remove this line

File: src/pages/dashboard/index.tsx [hunk 5]
```diff
@@ -84,8 +84,10 @@ const Dashboard = ({ user, isAuthenticated, myBriefs, myApplicationsResponse }:
 
   if (loadingStreamChat) return <FullScreenLoader />
 
+  console.log(user);
```
Comment (cuteolaf) [inline src/pages/dashboard/index.tsx:87]: remove this line

**Review by samelamin — APPROVED** [review]

**Pull request merged**